low-level/sspl-ll.spec

```diff
@@ -68,6 +68,7 @@ cp -afv files/opt/seagate/sspl/* $SSPL_BASE/
 # Rename setup directory to bin directory and remove .py extension of sspl_setup files.
 mv $SSPL_BASE/setup $SSPL_BASE/bin
 mv $SSPL_BASE/bin/sspl_setup.py $SSPL_BASE/bin/sspl_setup
+mv $SSPL_BASE/bin/sspl_post_install.py $SSPL_BASE/bin/sspl_post_install
```

Comment:
This is not needed as per my understanding. @ujjwalpl  to confirm! Basically its ok to deploy sspl_post_install.py i.e with py extension

Reply:
Done

Reply:
Done